packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -331,6 +331,16 @@ const COLUMNS = [
     name: _('vifNetworkLabel'),
     sortCriteria: (vif, userData) => userData.networks[vif.$network].name_label,
   },
+  {
+    itemRenderer: vif => (
+      <Number
+        onChange={rateLimit => setVif(vif, { rateLimit })}
+        value={vif.rateLimit || ''}
```

Comment:
How do we set it back to `undefined`? What does `undefined` mean? No limit?

Reply:
`0` or `undefined` will be an empty field which means no limit.
We can't back to `undefined`.  What's the problem if we can't back to `undefined`?

Reply:
- We need to be able to type `0` in the `Number` input
- What does the `0` value mean? No limit as well?